bindings/protocols/modbus/index.template.html

```diff
@@ -337,13 +350,12 @@ <h2>Examples</h2>
         default for observing a property. 
         <pre  id="example-polling" class="example" title="Polling">
             {
-                "href": "modbus+tcp://127.0.0.1:60000/1",
+                "href": "modbus+tcp://127.0.0.1:60000/1/1",
                 "op": [
                     "observeproperty"
                 ],
                 "modbus:entity": "Coil",
                 "modbus:pollingTime": 1000,
```

Comment:
I am not sure a single device can define the polling time. For RTU it depends on on the number of devices on the cable and the baud rate. Hence, it should not be mobus vocabulary.

The underlying problem is independent from the transport. Didn't we have "stability" or "frequency" to indicate how often a value is expected to change?